ncm-network/src/main/perl/network.pm

```diff
@@ -1694,6 +1695,36 @@ sub down_rename_devices
     return $action;
 }
 
+# Prepare and take the action for all daemons as defined in hash-reference C<$daemons>.
```

Comment:
i would like to see this moved (and perhaps generalised a bit) to `CAF::Service`. then we can use it in even more components. replace `$self` with `$logger` and export the function there

Reply:
Me too, are you volunteering? :smile_cat: